### ISSUE TITLE
bold font after \n in relationships text

### ISSUE THREAD

**Issue by dgutson** [plantuml/plantuml-stdlib#62]
Given
```
@startuml
title SS - Container1 - Components

top to bottom direction

skinparam {
  shadowing false
  arrowFontSize 10
  defaultTextAlignment center
  wrapWidth 200
  maxMessageSize 100
}

hide stereotype

skinparam rectangle<<SS.Container1.Component11>> {
  BackgroundColor #85bbf0
  FontColor #000000
  BorderColor #5d82a8
  roundCorner 20
}
skinparam rectangle<<SS.Container1.Component12>> {
  BackgroundColor #85bbf0
  FontColor #000000
  BorderColor #5d82a8
  roundCorner 20
}
skinparam rectangle<<SS.Container2>> {
  BackgroundColor #438dd5
  FontColor #ffffff
  BorderColor #2e6295
  roundCorner 20
}

rectangle "==Container2\n<size:10>[Container]</size>" <<SS.Container2>> as SS.Container2

package "Container1\n[Container]" <<SS.Container1>> {
  skinparam PackageBorderColor<<SS.Container1>> #444444
  skinparam PackageFontColor<<SS.Container1>> #444444

  rectangle "==Component11\n<size:10>[Component]</size>" <<SS.Container1.Component11>> as SS.Container1.Component11
  rectangle "==Component12\n<size:10>[Component]</size>" <<SS.Container1.Component12>> as SS.Container1.Component12
}

SS.Container1.Component12 .[#707070,thickness=2].> SS.Container2 : "<color:#707070>Relation2"
SS.Container1.Component11 .[#707070,thickness=2].> SS.Container1.Component12 : "<color:#707070>one\ntwo"
SS.Container1.Component11 .[#707070,thickness=2].> SS.Container2 : "<color:#707070>Relation1"
@enduml
```

Output is:
![image](https://user-images.githubusercontent.com/9671770/199841264-73a98798-fc9e-4cc5-a758-5ef9938a8503.png)

Note that `two` is darker than `one`.

**Comment by dgutson** [comment]
Not sure if related to #61

**Comment by Potherca** [comment]
When the given example  is reduced to a [Short, Self Contained, Correct example](http://www.sscce.org/):

```plantuml
@startuml
(A) -> (B): "one\ntwo"
(C) -> (D): "<b>one\ntwo"
(E) -> (F) : "<color:#707070>one\ntwo"
@enduml
```

![](https://www.plantuml.com/plantuml/png/SoWkIImgAStDuT9mr5JGjLFGSDAqKb3ApqkDoIifprVYqd262Bk0X6sIx91aN24oRfeA8Ada_9py8YjbSmCGH5Bca9gN0ZGV0000)

One can see that the difference is not the font _weight_ (i.e. "bold") but the font _color_ (i.e. "black").

As far as I am aware, the font style reverting to default after a newline/linebreak is expected behaviour.

(I could be wrong, that is for @arnaudroques to decide).

The solution for the second line also being colored `#707070` is to declare the color again for that line:

```plantuml
@startuml
(A) -> (B) : "<color:#707070>one\n<color:#707070>two"
@enduml
```
![](https://www.plantuml.com/plantuml/png/SoWkIImgAStDuT9mr5JGjLFGSD9Ki59GiadEpyalib8sDm11k_oyr9WyDB6IydmbBeVKl1IWRW00)

Or using a stereotype:

```plantuml
@startuml
skinparam arrowFontColor<<gray>> #707070

(A) -> (B)<<gray>> : "one\ntwo"
@enduml
```

![](https://www.plantuml.com/plantuml/png/SoWkIImgAStDuIhEpimhI2nAp5L8B2hABtVBpojnpi_9BxAnIIzAhBIpKr0sDm11BYuDHqq5NJi53ITDk9oLWb9-NcfCNabvlXANGsfU2j140000)

**Comment by arnaudroques** [comment]
> As far as I am aware, the font style reverting to default after a newline/linebreak is expected behaviour.
> 
> (I could be wrong, that is for @arnaudroques to decide).

You are right, that's the point. The font style is reverting to default after a newline.
I realize that is not a standard behaviour. The initial intention (10 years ago?) was to keep syntax as short as possible so that people that actually type diagrams on keyboard do not have to close tags.
The implementation was also simpler.

Now changing this would have too many impacts so I'm afraid we have to live with it.
I hope this is not a major issue, since the workaround is, as you said, to repeat color definition.

**Comment by dgutson** [comment]
Thanks folks for the explanation, rhe workaround, (and the kink for self contained.. :) ).
I must admit that this is completely unexpected for the terrestrial user, which unless comes here and reports this and have nice maintainers to explain, it would have been never solved.

May I propose a function call to disable this behavior? Like `HIDE_LEGEND()` but for this one, and have this documented somewhere?
Maybe something like `KEEP_STYLE_ON_BREAKS()`
That would help colpatibility, avoid the verbosity of repeating the tag after each \n (or \l or \r, all things I tried).

**Comment by dgutson** [comment]
BTW CC @simonbrowndotje because I think the workaround should be implemented in the exporter of structurizr.
